bundle/bin/rke2-uninstall.sh

```diff
@@ -30,6 +34,13 @@ uninstall_disable_services()
 
 uninstall_remove_files()
 {
+    if [ -r /etc/redhat-release ] || [ -r /etc/centos-release ] || [ -r /etc/oracle-release ]; then
+        rm -f /etc/yum.repos.d/rancher-rke2*.repo
+        rm -f /etc/sysconfig/rke2-server
```

Comment:
Will this file only exist on RPM-based systems?

Reply:
Looks like it. Not on Ubuntu.

Reply:
Yes, only on RPM-based systems as it's created by the RPM: https://github.com/rancher/rke2-packaging/blob/master/rpm/centos7/server/rke2-server.spec#L31